### PR TITLE
Including token separator in the EDM public documentation

### PR DESCRIPTION
**Pull request by madalenamedo**
The information was shared by the Product Group in the following issue:
https://o365exchange.visualstudio.com/DefaultCollection/IP%20Engineering/_workitems/edit/2891536

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/sit-get-started-exact-data-match-create-schema.md [hunk 1]
```diff
@@ -139,6 +139,28 @@ The `ignoredDelimiters` flag doesn't support:
...
+- \.
+- \;
+- \?
+- \!
+- \\r
+- \\n  
+
+When you include a *token separator*, EDM will break the token where the separator is. For example, EDM will see the value **Middle-Last Name** into **Middle-Last** and **Name** for the `LastName` field. If the *ignoredDelimiters* is included for the `LastName` field with the character '-', that action only happens after the value is broken. In the end, EDM would see the following values **MiddleLast** and **Name**.
+
+To use the following characters as *ignoredDelimiters* and not *token separators*, a SIT that matches the corresponding format needs to be associated with the field. For the example, a SIT that detects multi-word string with dashes in it needed to be associated with the `LastName` field.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sit-get-started-exact-data-match-create-schema.md]:
```suggestion
To use the following characters as *ignoredDelimiters* and not *token separators*, a SIT that matches the corresponding format needs to be associated with the field. For example, a SIT that detects a multi-word string with dashes in it needs to be associated with the `LastName` field.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @madalenamedo, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by madalenamedo** [comment]
@JohanFreelancer9 I have committed, thank you for your help!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrfox** [comment]
asking for tech accuracy review from PG

**Pull request merged**